encoding/legacy-mb-schinese/gbk/dummy.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Untitled Document</title>
+</head>
+
+<body>
+nothing to see here
+move along
+</body>
+</html>
```

Comment:
It would probably have been a better idea, but i'd prefer to leave as is for now because i don't think i can afford the time to make the necessary changes.